src/App.tsx

```diff
@@ -1,20 +1,33 @@
-import { PeoplePage } from './components/PeoplePage';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import { Navbar } from './components/Navbar';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { PageNotFound } from './components/PageNotFound';
 
 import './App.scss';
 
-export const App = () => {
-  return (
-    <div data-cy="app">
-      <Navbar />
+const navList = [
+  {
+    href: '/',
+    title: 'Home',
+  },
+  {
+    href: '/people',
+    title: 'People',
+  },
+];
 
-      <div className="section">
-        <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
-        </div>
-      </div>
-    </div>
-  );
-};
+export const App = () => (
+  <div className="App" data-cy="app">
+    <Navbar navList={navList} />
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+      <Route path="/home" element={<Navigate to="/" />} />
+      <Route path="people">
+        <Route index element={<PeoplePage />} />
+        <Route path=":slug" element={<PeoplePage />} />
+      </Route>
+      <Route path="*" element={<PageNotFound />} />
+    </Routes>
```

Comment:
Use `Outlet` instead and move routes to a separate component
https://reactrouter.com/en/main/components/outlet